.github/workflows/test_common.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.10.x"
+          python-version: "3.11.x"
```

Comment:
is it possible to define a matrix of python versions only on ubuntu? and all other oses to run on 3.10 still? if not we can keep this and solve in some other place